recipes-support/simple-http-server/simple-http-server.bb

```diff
@@ -0,0 +1,22 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "GPLv2"
+LICENSE_PATH = "${S}"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0;md5=801f80980d171dd6425610833a22dbe6"
+inherit systemd
+
+SRC_URI += "file://simple-http-server.sh"
+SRC_URI += "file://simple-http-server.service"
+
+FILES_${PN} += "${sbindir}/simple-http-server.sh"
+
+do_install() {
+        install -d ${D}${sysconfdir}/hosted_files
+        install -d ${D}${systemd_unitdir}/system
+        install -d ${D}${sbindir}
+        install -m 0644 ${WORKDIR}/simple-http-server.service ${D}${systemd_unitdir}/system
+        install -m 0755 ${WORKDIR}/simple-http-server.sh ${D}${sbindir}
+
+}
+
+SYSTEMD_SERVICE_${PN} += "simple-http-server.service"
```

Comment:
If socket is opened by .socket unit file, simple http server will try to bind the same socket again and therefore, fail with the error : Address already in use.